### DISCUSSION TITLE
Your demos are too good!

### DISCUSSION THREAD

**Discussion by xgpt** [Textualize/textual#1939] [Ideas]
Please add a note or something that highlights that they are running in a terminal, this is not at all apparent to ANYONE who isn't using a Mac (or even someone like me, who has been using a Mac for less than a month!)

I'm not just trying to flatter here, although this project looks *beautiful* and I cannot wait to integrate it into something soon just so I can work with it.

I'm genuinely recommending that your demos highlight "This is a terminal" "This is a terminal running textualize"

It looks like HTML+CSS in a browser. Seriously smooth work y'all.

**Comment by davep** [comment]
It is mentioned right up front, [on the front page of the documentation](https://textual.textualize.io/), in the second paragraph:

> Build sophisticated user interfaces with a simple Python API. Run your apps **in the terminal** and (coming soon) a web browser.
